sideboard/config.py

```diff
@@ -9,14 +10,72 @@
 from validate import Validator
 
 
+_defaults_re = re.compile(r'(.+)-defaults(\.\w+)$')
+
+
+def uniquify(x):
+    """
+    Returns an ordered copy of `x` with duplicate items removed.
+
+    >>> uniquify(['a', 'b', 'a', 'c', 'a', 'd', 'a', 'e'])
+    ['a', 'b', 'c', 'd', 'e']
+
+    """
+    seen = set()
+    return [i for i in x if i not in seen and not seen.add(i)]
```

Comment:
Yeah, I figured a possibility might be that the function was working as intended but the docstring just implied something different. Updating the docstring is fine.

That's crazy that you can do that with list comprehension.